cypress/integration/todo_spec.js

```diff
@@ -55,13 +60,38 @@ describe('TodoMVC - React', () => {
   it('edits an existing todo', () => {
     addTodoItemInLocalStorage()
     todoItems()
-      .dblclick()
-      .find('.edit')
-      .clear()
-      .type(EDITED_ITEM)
-      .type('{enter}')
+    .dblclick()
+    .find('.edit')
+    .clear()
+    .type(EDITED_ITEM)
+    .type('{enter}')
 
     todoItems()
-      .should('contain', EDITED_ITEM)
+    .should('contain', EDITED_ITEM)
+  })
+
+  it('verify hint message', () => {
+    cy.get('input:first')
```

Comment:
I'd recommend to search it as `.new-todo`.
`input:first` locator can be easily broken and give us incorrect element